docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -0,0 +1,246 @@
+# Watch Face Complication Provider
+
+The watch face complication provider application is a service application to provide complication data.
+Complication provider application provides only data. The watch face application decides how to display the complications.
+The complication provider application provides the complication data by using the watch face complication provider API to the watch face application.
+ 
+## Prerequisites
+
+To enable your application to use the watch face complication functionality:
+
+1. To use the Watch Face Complication API to communicate with the watch face application, this application has to request permission by adding
+the following privilege to the `tizen-manifest.xml` file:
+
+    ```xml
+    <privileges>
+    	<privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types for the Watch Face Complication Provider API, include the `<watchface-complication-provider.h>` header file in your application:
+
+    ```cpp
+    #include <watchface-complication-provider.h>
+    ```
+
+## Updating Complication Data
+
+When the request occurs to update the complication data, the complication provider application can receive the request by adding `watchface_complication_provider_update_requested_cb()`. The watch face application can send the update request and this request also occurs after the complication provider application notifies that the complication data is changed.

```

Comment:
he watch face application can send the update request, and this request also occurs after the complication provider application notifies that the complication data is changed.